zk/datadog_checks/zk/zk.py

```diff
@@ -220,33 +230,51 @@ def report_instance_mode(self, mode):
             gauge_name = 'zookeeper.instances.%s' % k
             self.gauge(gauge_name, v)
 
-    def _send_command(self, command):
-        buf = StringIO()
+    @staticmethod
```

Comment:
Pulled out the logic for sending commands to the socket and made a static method, not sure if this is appropriate or avoided.